### PR TITLE
release 0.2.1

### PR DESCRIPTION
**Pull request by laka3000**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.1] - 2021-03-31
```
Comment (DZariusz) [inline CHANGELOG.md:8]: this is not hotfix, so version should be 0.3.0

Comment (DZariusz) [reply CHANGELOG.md:8]: I think you need to recreate branch

**Review by DZariusz — APPROVED** [review]

File: src/components/ui/HeaderContent/ContractButton.js [hunk 2]
```diff
@@ -11,19 +15,45 @@ const propTypes = {
...
         <Text size="xsmall" textAlign="end" weight={300} color="text-weak">
           Contract - [
           <Anchor
             size="xsmall"
+            disabled={!info.chainContractAddress}
             target="_blank"
             rel="noopener noreferrer"
-            href={`${process.env.REACT_APP_SCAN_URL}/${address}#readContract`}
-            label={truncate(address)}
+            href={`${process.env.REACT_APP_SCAN_URL}/${info.chainContractAddress}#readContract`}
```
Comment (DZariusz) [inline src/components/ui/HeaderContent/ContractButton.js:55]:
```suggestion
            href={`${process.env.REACT_APP_SCAN_URL}/${info.chainContractAddress}`}
```

File: CHANGELOG.md (hunk 1 above)
Comment (laka3000) [reply CHANGELOG.md:8]: gotcha, my bad

thanks

**Pull request closed without merging**